libcxx/test/libcxx/utilities/expected/expected.expected/no_unique_address.compile.pass.cpp

```diff
@@ -12,7 +12,10 @@
 
```

Comment:
Not attached to this file: can you please create an issue to describe what this patch is fixing? #68552 is technically fixed by PR #68733, so it would be nice to have a separate issue just to track this one.

Reply:
Done: <https://github.com/llvm/llvm-project/issues/70494>